packages/react/src/components/ChoiceGroup/ChoiceGroup.base.tsx

```diff
@@ -41,9 +40,8 @@ const focusSelectedOption = (options: IChoiceGroupOption[], keyChecked: string |
   }
 };
 
-// Test if focus came from a sibling DOM element
-const focusFromSibling = (evt: React.FocusEvent<HTMLElement>): boolean => {
-  return !!(evt.relatedTarget && !elementContains(evt.currentTarget, evt.relatedTarget as HTMLElement));
+const focusFromFocusTrapZone = (evt: React.FocusEvent<HTMLElement>): boolean => {
```

Comment:
I half-wonder if it would be better to add something like reusable a `data-focus-zone-ignore` attribute to ChoiceGroup that FocusTrapZone would pay attention to, rather than put focuszone-specific code in ChoiceGroup.

Either way, this works though!

Reply:
I opted to have it here since the issue only manifests in ChoiceGroup. I agree that having the focus management system handle it is a better overall design but with our work moving to vNext I felt this works. I suspect we'll have a similar issue in vNext with tabster and, assuming that's the case, it will be good to explore how to handle this type of scenario there.